packages/bpk-mixins/src/mixins/_buttons.scss

```diff
@@ -337,6 +337,7 @@
 ///   }
 
 @mixin bpk-button--featured {
+  background-image: linear-gradient(-180deg, $bpk-button-featured-gradient-start-color 0%, $bpk-button-featured-gradient-end-color 100%);
```

Comment:
Can we deprecate `bpk-button-featured-gradient-start-color` and `bpk-button-featured-gradient-end-color`, and create a new token for `bpk-button-featured-color`